docs/reference/gettingstartedexample/gse-debug-guide.md

```diff
@@ -0,0 +1,242 @@
+
+### How to debug FIFO overflow errors (-52012)
+The Getting Started Examples use many FIFOs to pass data and metadata between the Host and the FPGA. While many of these FIFOs are small or transmit small amounts of data, any of them can fail as you approach your total system bandwidth limitations.  Below are the FIFOs you can find the in the Acqusition, Tap, and Generation examples and links to other debugging steps to take when encountering these errors.
+
+-  Acquisition/ TAP FIFOs
+    - LLP Packets FIFO - There is one of these FIFOs per serial channel. These can overflow if the FIFO sending data from the FPGA to the host does not have any space in it. This can happen when we do not write LLP packets to disk on the host side fast enough and make space in the FIFO for incoming data. For a more detailed explanation refer to [How to debug Acquisition FIFO overflow](#how-to-debug-acquisition-fifo-overflow).
+
+- Common FIFOs - These FIFOs are small and do not move much data to the Host. If one of these FIFOs overflows, it is usually indicative a system bandwidth limitation. If one of these FIFOs fail, try following steps from other troubleshooting guides below. 
+    - Image Data FIFO - There are two of these FPGA to Host FIFOs by default in each GSE. TODO link - How to debug image display that does not show images or has a low frame rate 
+    - Image Metadata FIFO - There are two of these FPGA to Host FIFOs by default in each GSE. TODO link - How to debug image display that does not show images or has a low frame rate 
+    - I2C Timestamp FIFO - There is one of these FPGA to Host FIFOs by default in each GSE.
+    - GPIO Timestamp FIFO - There is one of these FPGA to Host FIFOs by default in each GSE.
+
+- Generation FIFOs - It should not be possible to see a FIFO overflow on the Host to FPGA LLP Packets FIFOs. The generation GSE holds off on writing data to the FIFO until the FIFO has space available. Refer to [How to debug Generation GSE packet timing errors](#how-to-debug-generation-gse-packet-timing-errors) for more Generation GSE specific debugging.
+
+### How to debug Generation GSE packet timing errors
+The first thing to check is the Serial Output Channel Status indicator. You should verify that the bytes going in and coming out of the DRAM match. If the DRAM manager is overflowing, it means the number of packets being sent out the serial output channel is less than then packets being sent from the host. Verify the TDMS data set has timestamps running at the correct rate.
+- TODO: Add link to the 'serial channel status' instructions below
+ 
+If you cannot figure out what part of the serial output data path is causing the timing errors, or are seeing FIFO overflow errors, take these steps to reduce system bandwidth. You can start by refering to gse-gen-common.md#reducing-system-bandwidth-usage. Afterwards, try some of these tasks to see which part of your system or data set is responsible for causing overflows.
```

Comment:
I suggest the following updates to the sentence, "You can start by refering to gse-gen-common.md#reducing-system-bandwidth-usage."

Refer to [PXIe-148X Getting Started Example - Common Generation Tutorials](./gse-gen-common.md#reducing-system-bandwidth-usage).